maxim/logger/together/client.py

```
@@ -0,0 +1,224 @@
+                
+        except Exception as e:
+            scribe().warning(
+                f"[MaximSDK][TogetherInstrumentation] Error in async streaming generation: {e}"
+            )
+
+    def wrap_sync_create(create_func):
+        '''Wrapping the create function of chat completion for the synchronous client'''
+
+        @functools.wraps(create_func)
+        def wrapper(self: ChatCompletions, *args: Any, **kwargs: Any):
+            extra_headers = kwargs.get("extra_headers", None)
```

Ruff (0.11.9)

Use `kwargs.get("extra_headers")` instead of `kwargs.get("extra_headers", None)`

Replace `kwargs.get("extra_headers", None)` with `kwargs.get("extra_headers")`

(SIM910)